### PR TITLE
Move MIME and HTTP unit tests under respective modules

### PR DESCRIPTION
**Pull request by arukshani**
$subject

### CODE REVIEW

File: stdlib/mime/src/test/java/org/ballerinalang/stdlib/mime/Util.java [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+import static org.ballerinalang.mime.util.MimeConstants.MULTIPART_MIXED;
+import static org.ballerinalang.mime.util.MimeConstants.OCTET_STREAM;
+import static org.ballerinalang.mime.util.MimeConstants.PROTOCOL_PACKAGE_IO;
+import static org.ballerinalang.mime.util.MimeConstants.READABLE_BYTE_CHANNEL_STRUCT;
+import static org.ballerinalang.mime.util.MimeConstants.TEXT_PLAIN;
+
+/**
+ * Contains utility functions used by mime test cases.
+ *
+ * @since 0.963.0
```
Comment (daneshk) [inline stdlib/mime/src/test/java/org/ballerinalang/stdlib/mime/Util.java]: Shall we change the version to the new release

File: stdlib/mime/src/test/resources/datafiles/file.xml [hunk 2]
```diff
@@ -0,0 +1 @@
+<name>Ballerina xml file part</name>
```
Comment (daneshk) [inline stdlib/mime/src/test/resources/datafiles/file.xml]: shall we add new line at EOF

File: stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/TestCallableUnitCallback.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        this.request = request;
+    }
+
+//    @Override
+//    public void notifyReply(BValue... response) {
+//        //TODO check below line
+//        HttpCarbonMessage responseMessage = HttpUtil.getCarbonMsg((BStruct) response[0], null);
+//        this.responseMsg = responseMessage;
+//        this.executionWaitSem.release();
+//    }
```
Comment (daneshk) [inline stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/TestCallableUnitCallback.java]: Shall we remove commented code

File: stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/Services.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.Map;
+
+import static org.ballerinalang.net.http.HttpConstants.SERVICE_ENDPOINT_CONFIG;
+
+/**
+ * This contains test utils related to Ballerina service invocations.
+ *
+ * @since 0.8.0
```
Comment (daneshk) [inline stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/Services.java]: Shall we change the version to the next release

File: stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/MessageUtils.java [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+import java.net.InetSocketAddress;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * RequestResponseUtil Class contains method for generating a message.
+ *
+ * @since 0.8.0
```
Comment (daneshk) [inline stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/MessageUtils.java]: Shall we change the version to the next release

File: stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/TestCallableUnitCallback.java (hunk 3 above)
Comment (arukshani) [reply stdlib/http/src/test/java/org/ballerinalang/stdlib/utils/TestCallableUnitCallback.java]: This was commented in the original code as well. I only moved the code from unit test to the relevant module.

**Pull request merged**